code/game/mecha/working/clarke.dm

```diff
@@ -48,6 +48,24 @@
 		var/mob/living/brain/B = M.brainmob
 		hud.add_hud_to(B)
 
+/obj/mecha/working/clarke/play_stepsound(direction)
+	if(prob(30))
+		floor_damage()
+	. = ..()
+
+/obj/mecha/working/clarke/mechturn(direction)
+	if(prob(80))
+		floor_damage()
+	. = ..()
+
+/obj/mecha/working/clarke/proc/floor_damage()
+	var/turf/open/floor/tile = get_turf(src)
```

Comment:
why not make it check for "plasteel" floors

Reply:
Because it should also tear up wooden flooring, carpets, grass, etc.

Reply:
As someone that has worked on tracked vehicles for years, yes, they would tear up wooden flooring, carpets, grass, etc. 
I have no problem with any of that. Tracks fuck shit up.

Reply:
thank you for your service